apps/calendar/components/calendar/week/WeekColumnsList.tsx

```diff
@@ -143,20 +147,23 @@ const WeekColumnsList = () => {
         if (joinedPerson?.includes(lead.id)) return lead;
       });
     }
-
+//Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.
     if (cellContext.filterBy?.length && cellContext.filterBy.length > 0) {
       leads = leads?.filter((lead) => cellContext.filterBy?.includes(lead?.subsystem || ""));
     }
+    //Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.
```

Comment:
//Break down the logic into smaller functions or separate utility functions to improve code readability and reusability.